### PR TITLE
support k8s outdated api db

### PR DESCRIPTION
**Pull request by chen-keinan**
support k8s outdated api db
Related: #233 231

### CODE REVIEW

File: pkg/k8s/api/outdated.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+	}
+
+	return nil
+}
+
+func (vs Outdated) save(advisories []OutdatedApiDb) error {
+	err := vs.dbc.BatchUpdate(func(tx *bolt.Tx) error {
+		for _, adv := range advisories {
+			if err := vs.dbc.PutK8sDb(tx, dataType, adv); err != nil {
+				return xerrors.Errorf("failed to put data source: %w", err)
```
Comment (knqyf263) [inline pkg/k8s/api/outdated.go:72]: Is it a data source? Seems like just data about outdated API

File: pkg/k8s/api/outdated.go [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+type OutdatedApiDb types.OutDatedAPIData
+
+var (
+	dataType = "outdated-api"
+
+	source = types.DataSource{
+		ID:   K8sOutdatedAPI,
+		Name: "Kubernetes GitHub docs",
+		URL:  "https://github.com/kubernetes/kubernetes",
+	}
```
Comment (knqyf263) [inline pkg/k8s/api/outdated.go:29]: Do we want to keep it? It is not used.

**Review by knqyf263 — COMMENTED** [review]
Looks good! I left nit comments.

File: pkg/k8s/api/outdated.go (hunk 1 above)
Comment (knqyf263) [reply pkg/k8s/api/outdated.go:72]:
```suggestion
				return xerrors.Errorf("failed to put outdated API data: %w", err)
```

File: pkg/k8s/api/outdated.go (hunk 2 above)
Comment (chen-keinan) [reply pkg/k8s/api/outdated.go:29]: I will remove it

**Pull request closed without merging**